Sources/FingerprintJS/Harvesters/DataExchange/DeviceInfo.swift

```diff
@@ -16,6 +16,8 @@ public struct DeviceInfo {
     public let physicalMemory: String?
     public let cpuCount: String?
 
+    /// The operating system's current time zone.
+    public let osTimeZone: TimeZone
```

Comment:
Do we want to introduce an external dependency on Apple's types? My immediate thought would be to replace this with either some primitive type (e.g. it could be just a String with the TimeZone identifier) or introducing a layer of indirection that would let us control the type in the future.

Reply:
In this particular case, the use of `TimeZone` type won't introduce any additional dependencies to Cocoa Touch frameworks, since we already use the type `UUID` that also comes from `Foundation` framework and then there are also `CGSize` and `CGFloat` from `CoreFoundation`. But whether we should expose an instance of `TimeZone` or a simple `String` representing time zone identifier remains an open question and to me the decision on which approach to use depends on the purpose of `DeviceInfo` struct. Let's discuss it.

Reply:
As agreed, the main idea behind `DeviceInfo` struct is to provide a simple interface for interoperability with other languages/technologies. For this reason, we decided to ditch the `TimeZone` type in favor of `String` type and rename the property to `osTimeZoneIdentifier`.